__tests__/regression/fix-1813.test.js

```
@@ -12,7 +13,10 @@ import {
   getButtons,
   getButtonsLength,
   getClonesCount,
-  getSlidesCount
+  getCurrentSlide,
```

ESLint

[error] 'getCurrentSlide' is defined but never used.

(no-unused-vars)